### ISSUE TITLE
BRDF & Albedo

### ISSUE THREAD

**Issue by ShunkaiZhou** [tyz1030/darkgs#3]
Hi Zhang, thank you for your great work. I have a question about the paper in ArXiv.
May I ask why you multiply brdf and albedo?
<img width="236" alt="brdf" src="https://github.com/tyz1030/darkgs/assets/124686454/fe3f83c1-3baa-49fd-ad5c-1b15c724e0e1">

**Comment by tyz1030** [comment]
Yes, you are right. Thanks a lot for pointing it out, I will revise my arxiv. While the final equations are correct, $c$ should be (in) the BRDF.

When writing the paper I intended to merge the lambert cosine term into BRDF for simplicity, following the convention in NeRV paper.

Let me know if it doesn't make sense to you or you find anything else confusing. I really appreciate your feedback!

**Comment by ShunkaiZhou** [comment]
OK! I understand and thank you very much.
